wandb/sdk/launch/create_job.py

```diff
@@ -206,6 +208,14 @@ def _create_job(
     if "latest" not in aliases:
         aliases += ["latest"]
 
+    if schema:
+        _validate_schema(schema)
+        schema = {
```

Comment:
nit: this variable would be better named `metadata`, since it's a different structure than `schema` and is assigned to a parameter called `metadata`